Documentation/devel/stage1-implementors-guide.md

```diff
@@ -57,9 +61,10 @@ An alternative stage 1 would need to do whatever is appropriate for entering the
 
 #### Arguments
 
-1. image id of the specific application to enter
-2. cmd to execute
-3. any cmd arguments
+1. pid of the container's PID 1. rkt finds that pid by one of the two supported methods described in the `rkt run` section.
```

Comment:
"PID of the process that is PID 1 in the container"

Also, in general you use "PID" (uppercase) when you refer to "PID 1" and "pid" otherwise, I think you should always use PID and remove that distinction